src/InitializerInterface.php

```diff
@@ -29,8 +29,7 @@ interface InitializerInterface extends Initializer\InitializerInterface
     /**
      * Initialize
      *
-     * @param mixed $instance
      * @return mixed
      */
-    public function initialize($instance, ServiceLocatorInterface $serviceLocator);
+    public function initialize(mixed $instance, ServiceLocatorInterface $serviceLocator);
```

Comment:
Can we add this to interfaces? Seems unsafe 🤔

Reply:
It already in `PluginManagerInterface` as well, see 

https://github.com/laminas/laminas-servicemanager/pull/157/files#diff-891bff41f1394c816120c19c5f4f7cfc98fa92047c8efae9b4ccd9f51ea83eae